docs/README.limitations.md

```diff
@@ -0,0 +1,16 @@
+# Known Limitations
+
+This document highlights the current limitations when using secrets-store-csi-driver.
+
+## Mounted content and Kubernetes Secret not updated after secret is updated in external secrets-store
+
+When the secret/key is updated in external secrets store after the inital pod deployment, the updated secret is not automatically reflected in the pod mount or the Kubernetes secret. 
+
+This feature is planned for release (`v0.0.15`). See [design doc](https://docs.google.com/document/d/1RGT0vmeUnN71n_u5fZKsSCa2YQpGw99rfGN9RlFMgHs/edit?usp=sharing) for more details.
+
+### How to fetch the latest content with the current releases?
```

Comment:
Can we say `fetch the latest content with release v0.0.14 and earlier`

Reply:
Updated!